cli/cmd/project/connect.go

```diff
@@ -163,6 +164,7 @@ func ConnectCmd(cfg *config.Config) *cobra.Command {
 	connectCmd.Flags().StringVar(&description, "description", "", "Project description")
 	connectCmd.Flags().BoolVar(&public, "public", false, "Make dashboards publicly accessible")
 	connectCmd.Flags().StringSliceVarP(&variables, "env", "e", []string{}, "Set project variables")
+	connectCmd.Flags().StringVar(&region, "region", "", "A region of the deployment. <cloud provider>-<region>")
```

Comment:
Make the helper `"Deployment region"`. Let's not describe format as it can be anything in theory (people should get it from docs or something).